ui/js/util/getThemes.js

```diff
@@ -0,0 +1,20 @@
+const { readdirSync } = require("fs");
```

Comment:
So I _think_ it would be appropriate for this to handled via just `selector/settings.js` rather than it's own utility function.

Although technically reading from disk should be considered non-deterministic, which would mean this would need to have it's own action and reducer. But that seems somewhat over-engineered to me ( @6ea86b96 would you do that here? )

Another option would be to have the build process put the themes into `app/package.json`, which there is a precedence for importing configuration values from.